### PR TITLE
add check on empty strings to prevent StringIndexOutOfBoundsException

### PR DESCRIPTION
**Pull request by debbeca**

### CODE REVIEW

File: spring-boot-2/src/main/java/org/quickperf/web/spring/QuickPerfAfterRequestServletFilter.java [hunk 1]
```diff
@@ -199,8 +200,10 @@ private void quickPerfProcessing(HttpServletRequest httpServletRequest, HttpServ
             infoReport.append("* SQL");
             infoReport.append(lineSeparator());
             String sqlExecutionsAsString = sqlExecutions.toString();
-            String sqlExecutionsWithoutThreeLastLineBreaks = sqlExecutionsAsString.substring(0, sqlExecutionsAsString.length() - 3);
-            infoReport.append(sqlExecutionsWithoutThreeLastLineBreaks);
+            if(!StringUtils.isEmpty(sqlExecutionsAsString)) {
```
Comment (jeanbisutti) [inline spring-boot-2/src/main/java/org/quickperf/web/spring/QuickPerfAfterRequestServletFilter.java:203]: Perhaps we could move the check before` infoReport.append("* SQL");`?

**Review by jeanbisutti — COMMENTED** [review]
Thank you very much for having reported and fixed this problem!

Just one comment on the check.

File: spring-boot-2/src/main/java/org/quickperf/web/spring/QuickPerfAfterRequestServletFilter.java (hunk 1 above)
Comment (debbeca) [reply spring-boot-2/src/main/java/org/quickperf/web/spring/QuickPerfAfterRequestServletFilter.java:200]: I ve just put it in the beginning.

**Comment by jeanbisutti** [comment]
Thank you very much Walid!

**Pull request merged**